.github/workflows/pr-push.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
```

Comment:
I'm not terribly familiar with the workflows here (I always relied on W3C folks to set it up! 😨). Looking around, though, I see more `auto-publish.yml` than `pr-push.yml` (e.g. https://github.com/w3c/spec-prod/blob/main/docs/examples.md).

Reply:
Done.